src/formats/PDB.cpp

```diff
@@ -138,6 +142,17 @@ void PDBFormat::read(Frame& frame) {
             // Else we have read a frame
             got_end = true;
             continue;
+        case Record::HELIX:
+            read_HELIX(line);
+            continue;
+        case Record::SHEET:
+            secinfo_.emplace_back(std::make_tuple(line[21], std::stoul(line.substr(22, 4)),
```

Comment:
I would rather have a function to do the reading and handle (with an error) the case of lines that are not long enough, or that contain strings where a number is expected.